aea/manager/project.py

```diff
@@ -93,8 +94,9 @@ def load(
         cls,
         working_dir: str,
         public_id: PublicId,
-        is_local: bool = False,
-        is_remote: bool = False,
+        local: bool = False,
+        remote: bool = False,
+        ipfs: bool = False,
```

Comment:
IPFS is a form of remote registry.

This is how I see it:
local
   -> packages folder
   -> ipfs (e.g. the approach with the local file and local node)
remote
   -> http
   -> ipfs (e.g. the approach with a hosted node)

Reply:
In light of my response here (https://github.com/valory-xyz/open-aea/pull/102#discussion_r847632786) this would now be:

local
-> packages folder
remote
-> ipfs (default option)
-> http (can be set via init)